File: backend-module/eneity/Exam.java
@Data
public class Exam {
    private int id;
    private String name;
    private String type;
    private String semester;
    private int classId;
    private LocalDate examDate;
    private int isArchived;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/exam.ts
export interface Exam {
  id: number
  name: string
  type: string
  semester: string
  classId: number
  examDate: string | null
  isArchived: number
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface ExamForm {
  id?: number
  name: string
  type: string
  semester: string
  classId: number | null
  examDate: string | null
}
